code/core/src/preview-api/modules/preview-web/render/StoryRender.ts

```diff
@@ -272,16 +283,26 @@ export class StoryRender<TRenderer extends Renderer> implements Render<TRenderer
             context.mount = async () => {
               throw new MountMustBeDestructuredError({ playFunction: playFunction.toString() });
             };
-          }
-          await this.runPhase(abortSignal, 'playing', async () => {
+            await this.runPhase(abortSignal, 'playing', async () => playFunction(context));
+          } else {
+            // when mount is used the playing phase will start later, right after mount is called in the play function
             await playFunction(context);
-          });
+          }
+
+          if (!mounted) {
+            throw new NoStoryMountedError();
```

Comment:
Duplicate with https://github.com/storybookjs/storybook/pull/28431 .. were these PRs up split properly?

Reply:
sorry changed the base branch now